openshift/templates/dancer-mysql.json

```diff
@@ -330,6 +334,11 @@
       "value": "openshift/mysql-55-centos7"      
     },
     {
+      "name": "PERL_APACHE2_RELOAD",
+      "description": "Enables automatic reloading of Perl modules",
+      "value": "false"
```

Comment:
@mnagy put to the description what are the expected values(true, yes). Or if the hot deploy would be enabled only with the 'true' value, say it so in the description.